ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/OAuthRedirectExample/windows_deployment.bat

```diff
@@ -2,13 +2,13 @@ ECHO OFF
 
 SET "ARCH=x64"
 SET "VC_ARCH=amd64"
-SET "QT_KIT=C:\Qt\5.6\msvc2015_64"
+SET "QT_KIT=C:\Qt\5.12.0\msvc2017_64"
 SET "DIR=%~dp0"
 SET "BUILD_TYPE=release"
-SET "BUILD_DIR=%DIR%..\build-OAuthRedirectExample-Desktop_Qt_5_6_2_MSVC2015_64bit-Release\%BUILD_TYPE%"
-SET "ARCGIS_SDK_DIR=C:\Program Files (x86)\ArcGIS SDKs\Qt100.5\sdk\windows\%ARCH%\bin\%BUILD_TYPE%"
+SET "BUILD_DIR=%DIR%..\build-OAuthRedirectExample-Desktop_Qt_5_12_0_MSVC2017_64bit-Release\%BUILD_TYPE%"
+SET "ARCGIS_SDK_DIR=C:\Program Files (x86)\ArcGIS SDKs\Qt100.6\sdk\windows\%ARCH%\bin\%BUILD_TYPE%"
 
-call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" %VC_ARCH%
+call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Auxiliary\Build\vcvarsall.bat" %VC_ARCH%
```

Comment:
Good catch! Have these new paths been tested?

Reply:
I didn't test it locally by running the sample, but I did verify that these are the new, correct paths for VS 2017 now.